common/src/main/java/com/wynntils/features/user/QuestInfoOverlayFeature.java

```diff
@@ -48,24 +82,15 @@ public void render(PoseStack poseStack, float partialTicks, Window window) {
                 return;
             }
 
-            List<TextRenderTask> toRender = new ArrayList<>();
-
-            toRender.add(TextRenderTask.getWithHorizontalAlignment(
-                    "Tracked Quest Info:", this.getWidth(), CommonColors.GREEN, this.getRenderHorizontalAlignment()));
-            toRender.add(TextRenderTask.getWithHorizontalAlignment(
-                    currentQuest.quest(), this.getWidth(), CommonColors.ORANGE, this.getRenderHorizontalAlignment()));
-            toRender.add(TextRenderTask.getWithHorizontalAlignment(
-                    currentQuest.description(),
-                    this.getWidth(),
-                    CommonColors.WHITE,
-                    this.getRenderHorizontalAlignment()));
+            toRender[1].setText(currentQuest.quest());
```

Comment:
As a future possible enhancement, we can have the quest manager send out an even when the quest changes, so we need to set the text only once and not once per render. But we can skip it for now.

Reply:
I thought about that, and I don't think the cost of registering overlays into event bus, then listening for an event is much more optimized than setting a string reference.